### PR TITLE
fix(plugin-meetings): added userId, orgId and meetingId for upload logs

### PR DESCRIPTION
**Pull request by arun3528**
Fixes #[INSERT LINK TO ISSUE NUMBER]

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by adamweeks — CHANGES_REQUESTED** [review]
Can you split the commits into the proper areas? One for meetings, one for i-p-support?

**Comment by arun3528** [comment]
hey @adamweeks  whats the most easy way to do it

**Comment by adamweeks** [comment]
@arun3528 you can rebase your commit into two. Here's a good doc on it https://emmanuelbernard.com/blog/2014/04/14/split-a-commit-in-two-with-git/

**Comment by arun3528** [comment]
@adamweeks fixed the review comments

**Comment by adamweeks** [comment]
Looks like you lost the changes to the support plugin and [this commit](https://github.com/webex/webex-js-sdk/pull/2003/commits/b1059c8436b67878b25fa1db66b12e36fad6b7da) has changes to the meetings plugin even though the scope message says `support`

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 1]
```diff
@@ -685,6 +685,18 @@ export default class Meeting extends StatelessWebexPlugin {
         }
       );
 
+      // Upload logs on media inactivity
+      // Normally media should not be inactive
+      Trigger.trigger(
+        this,
+        {
+          file: 'meeting/index',
+          function: 'leave'
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:694]:
```suggestion
          function: 'setUpLocusInfoMediaInactiveListener'
```

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 2]
```diff
@@ -2549,6 +2561,17 @@ export default class Meeting extends StatelessWebexPlugin {
           }
         );
 
+        // Upload logs on join Failure
+        Trigger.trigger(
+          this,
+          {
+            file: 'meeting/index',
+            function: 'leave'
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:2569]:
```suggestion
            function: 'join'
```

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 3]
```diff
@@ -2992,6 +3015,17 @@ export default class Meeting extends StatelessWebexPlugin {
           }
         );
 
+        // Upload logs on error while adding media
+        Trigger.trigger(
+          this,
+          {
+            file: 'meeting/index',
+            function: 'leave'
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:3023]:
```suggestion
            function: 'addMedia'
```

**Review by adamweeks — APPROVED** [review]

**Pull request merged**